src/ducks/files/Container.jsx

```diff
@@ -18,6 +19,8 @@ import {
 const mapStateToProps = (state, ownProps) => ({
   isTrashContext: false,
   canUpload: true,
+  isRenaming: isRenaming(state),
```

Comment:
Not sure that `isRenaming` should be hosted in the global state. It may stay in a stateful component responsible of the file name edition.